services/backend/api_doc/coverages.js

```diff
@@ -37,3 +37,34 @@
  *    }
  *  ]
  */
+
+/**
+ * @apiDefine CoverageLayerInEAExample
```

Comment:
If you are going to use this example in every endpoint, you should name it `CoverageLayerInGeofenceExample`

Reply:
Done